docs/index.rst

```diff
@@ -1,10 +1,56 @@
-Welcome to the Software Factory documentation
-*********************************************
+Welcome to the PELUX Baseline Software Factory
+**********************************************
 Revision: |release|
 
 -------------------
 
-The Software Factory documentation is the central space where we collect all useful information on how to work within the project.
+What is the Software Factory?
+-----------------------------
+The Software Factory documentation contains processes, instructions and how-tos
+for how to work with PELUX, as well as instructions for how to create a
+standalone project based on PELUX.
+
+What is PELUX?
+--------------
+From pelux.io_:
+
+    *PELUX is a Linux based platform for your automotive infotainment project.
+    The PELUX software architecture is optimized to secure the shortest path
+    from silicon to pixel and thereby provide optimal performance to deliver a
+    stunning infotainment solution. It’s based on Yocto and offers an improved
+    developer experience.*
+
+
+Why PELUX?
+^^^^^^^^^^
+PELUX is a base platform for building automotive infotainment systems (IVI
+systems for short). IVI systems are usually responsible for everything shown in
```

Comment:
I know I'm late to the party, but why not write "In-Vehicle Infotainment systems" instead of "automotive infotainment systems"?

Reply:
I thought the latter would sound more user-friendly, but perhaps it is not.

Reply:
After I had written this I realized there is a previous sentence which uses the phrase "automotive infotainment systems". My issue with the sentence where I commented was that the phrase "for short" implies an abbreviation for me, but this isn't an abbreviation, it's another way of naming for the same type of system. It's not a big issue. I would have preferred a re-write for clarity. Maybe just "often called IVI systems" or something like that.